src/Fixer/Phpdoc/PhpdocListTypeFixer.php

```diff
@@ -65,6 +65,6 @@ public function getPriority(): int
 
     protected function normalize(string $type): string
     {
-        return Preg::replace('/array(?=<(?:[^,<]|<[^>]+>)+(>|{|\())/i', 'list', $type);
+        return Preg::replace('/(?<!\w)array(?=<(?:[^,<]|<[^>]+>)+(>|{|\())/i', 'list', $type);
```

Comment:
```suggestion
        return Preg::replace('/\barray(?=<(?:[^,<]|<[^>]+>)+(>|{|\())/i', 'list', $type);
```
This also works. I don't have any preference. Only argument I have is that `\b` is shorter. Do you have in mind any case where one would work and the other not?

Reply:
`\w` and `\b` should consider word of the same chars